README.md

```diff
@@ -1,75 +1,176 @@
 
-Run these from the same host where `oc` is logged into the OpenShift cluster.
-
-**NOTES:**
-- the default settings in the create script are tested on an openshift vagrant development vm, so you will likely have to overwrite some variables unless you are running these commands directly in the vagrant development vm.
+A simple example on labeling the nodes
 
 ```bash
-cd /mnt/src/fh-openshift-templates/scripts/
-./create-mbaas.sh
-```
 
-#### Sample commands with variables being overwritten
+oc label node mbaas-1 type=mbaas
+oc label node mbaas-2 type=mbaas
+oc label node mbaas-3 type=mbaas
 
-Run these from the same host where `oc` is logged into the OpenShift cluster.
+oc label node mbaas-1 mbaas_id=mbaas1
+oc label node mbaas-2 mbaas_id=mbaas2
+oc label node mbaas-3 mbaas_id=mbaas3
 
-```bash
-cd scripts/
-DOCKERCFG_FILE=~/.dockercfg MBAAS_TEMPLATE_FILE=./fh-mbaas-template-3node.json MBAAS_PROJECT_NAME=3node-mbaas ./create-mbaas.sh
 ```
 
-The output will include `events` and `pod` info, check the health endpoint of the MBaaS, and finish with a full `Nagios` check.
-If there are any issues with the installation, check the output for any errors.
+### 1 Node persistent MBaaS
 
-**NOTES:**
-- Any failing Nagios checks will **not** cause the script to fail. Instead, a WARNING will be output. This may change depending on the variation of supported templates and how the Nagios checks can be dynamically defined.
-- If installing the 1 node persistent MBaaS, and there are warnings or failures for `mongodb-2:Ping`, `mongodb-3:Ping` & `mongodb::replicaset` checks, these can be ignored.
-- It may take a while the first time an MBaaS is installed in your cluster while the docker images are pulled. This can manifest as `ImagePullBackOff` errors for pods after a few minutes. These can be ignored unless that error occurs very early in the script execution. In that case, it would suggest invalid docker hub credentials or the docker hub account doens't have access to the image(s).
+RHMAP 1-Node MBaaS for OpenShift will require the following resources outlined in the table below at a minimum:
 
-### Accessing the Nagios Dashboard
+- **Prerequisites**
+    - OpenShift project created 
+    - A 25G PV (persistent volume) needs to be available
+    - Acces to this repo (i.e clone the repository)
 
-Run this command to get the Nagios Dashboard url (with username & password)
 
-```bash
-echo "https://$(oc env dc/nagios --list|grep NAGIOS_USER | awk -F'=' '{print $2}'):$(oc env dc/nagios --list|grep NAGIOS_PASSWORD | awk -F'=' '{print $2}')@$(oc get route nagios --template "{{.spec.host}}")"
-```
 
-## Building
-```shell
-$ npm install
-```
+	|  Description                               | Parameter name               | Default value           | Fail/warning/recommended             |
```

Comment:
Tables with parameters are not very relevant here. Lets stick to listing prerequisites as a simple list.